src/amberscript/parser.cc

```diff
@@ -1884,6 +1906,7 @@ Result Parser::ParseRun() {
     cmd->SetTopology(topo);
     cmd->SetFirstVertexIndex(start_idx);
     cmd->SetVertexCount(count);
+    cmd->SetPolygonMode(pipeline->GetPolygonMode());
```

Comment:
Does the DrawGridCommand and DrawRectCommand also need to have the PolygonMode set to be consistent?

Reply:
Propably yes. I'll add it there too.